core-libs/setup/ssr/logger/loggers/default-server-logger.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LogContext, ServerLogger } from './server-logger';
+
+export class DefaultServerLogger extends ServerLogger {
+  log(message: string, context?: LogContext) {
+    /* eslint-disable-next-line no-console */
+    console.log(this.createLogMessage(message, context));
+  }
+  warn(message: string, context?: LogContext) {
+    /* eslint-disable-next-line no-console */
+    console.warn(this.createLogMessage(message, context));
+  }
+  error(message: string, context?: LogContext) {
+    /* eslint-disable-next-line no-console */
+    console.error(this.createLogMessage(message, context));
+  }
+
+  protected createLogMessage(message: string, context: LogContext | undefined) {
+    return {
+      message,
+      context: {
+        timestamp: new Date().toISOString(),
+        ...context,
+      },
+    };
+  }
+}
```

Comment:
[MEDIUM]

I guess we're lacking here the JSON.stringify() - comparing to `ExpressServerLogger`?